README.md

```diff
@@ -7,12 +7,7 @@ Standard stylelint config for Netzstrategen projects.
 Install the configuration and all peer dependencies:
 
 ```bash
-npm install --save-dev \
-  @netzstrategen/stylelint-config \
-  stylelint \
-  stylelint-order \
-  stylelint-scss \
-  stylelint-selector-bem-pattern
+npm install --save-dev @netzstrategen/stylelint-config stylelint
```

Comment:
I see `stylelint` as a peer dependency in https://github.com/stormwarning/stylelint-config-recess-order/blob/main/package.json#L62 so this doesn't need to be explicitly installed here, or?


> As of npm v7, peerDependencies are installed by default.

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependencies

Reply:
I added it as a peer dependency here because other packages in a project will likely depend on it. So gulp-task-collection depends on stylelint for example